transcendence-app/src/chat/chat.controller.ts

```diff
@@ -192,18 +193,19 @@ export class ChatController {
   @Get('room/member')
   @ApiOkResponse({
     description: `Lists all chatrooms the auth user is subscribed to (max ${MAX_ENTRIES_PER_PAGE})`,
-    type: [Chatroom],
+    type: [GenericChat],
   })
   @ApiBadRequestResponse({ description: 'Bad Request' })
   @ApiServiceUnavailableResponse({ description: 'Service unavailable' })
   async getAuthChatrooms(
     @GetUser() user: User,
     @Query() chatsPaginationQueryDto: PaginationWithSearchQueryDto,
-  ): Promise<Chatroom[]> {
-    const chatrooms = await this.chatService.retrieveChatroomsforAuthUser(
-      user,
-      chatsPaginationQueryDto,
-    );
+  ): Promise<GenericChat[]> {
+    const chatrooms =
+      await this.chatService.getAuthUserPaginatedChatsAndChatrooms(
```

Comment:
issue: This error shows up in the console when we paginate the chats:

<details>

```
Warning: Encountered two children with the same key, `27aaab4f-acd5-4152-8e6d-b1478ec86585`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
ul
RowsList@http://localhost:3000/static/js/bundle.js:11586:7
div
RowsListTemplate@http://localhost:3000/static/js/bundle.js:13219:7
div
MainTabPageTemplate@http://localhost:3000/static/js/bundle.js:13062:7
SearchContextProvider@http://localhost:3000/static/js/bundle.js:13572:7
div
div
ChatsPageTemplate@http://localhost:3000/static/js/bundle.js:2488:7
ChatsPage@http://localhost:3000/static/js/bundle.js:2300:70
RequireAuth@http://localhost:3000/static/js/bundle.js:11301:7
RenderedRoute@http://localhost:3000/static/js/bundle.js:58858:7
Routes@http://localhost:3000/static/js/bundle.js:59323:7
AppRoutes
UserBlocklistProvider@http://localhost:3000/static/js/bundle.js:13720:7
UserFriendProvider@http://localhost:3000/static/js/bundle.js:13840:7
UserStatusProvider@http://localhost:3000/static/js/bundle.js:13953:7
AuthProvider@http://localhost:3000/static/js/bundle.js:14081:7
NotificationContextProvider@http://localhost:3000/static/js/bundle.js:13400:7
AppContext@http://localhost:3000/static/js/bundle.js:728:7
Router@http://localhost:3000/static/js/bundle.js:59265:7
BrowserRouter@http://localhost:3000/static/js/bundle.js:57516:7
```

</details>

And if we search the duplicated chat name, it shows duplicated entries:

<details>

![image](https://user-images.githubusercontent.com/54190400/214566539-d40abe74-9184-4407-9fb1-d95816c8b08c.png)

</details>

Reply:
done. I've also removed the order by from the table I'm creating just for the one to one messages (the one I'm later merging), since one order by is enough.